testing/test_fot

```diff
@@ -76,30 +76,30 @@ vlan=$($YQ '.dps."nz-kiwi-t2sw1".interfaces."1".native_vlan' $BEHAVIORAL_CONFIG)
 echo $(( (vlan > 271) + (vlan > 276) )) | tee -a $TEST_RESULTS
 echo $(( (vlan > 271) + (vlan > 276) )) | tee -a $TEST_RESULTS
 
-monitor_daq_log "Send device result 9a:02:57:1e:8f:01 passed"
-monitor_daq_log "Send device result 9a:02:57:1e:8f:02 failed"
-monitor_daq_log "Target device 9a02571e8f03 test hold running"
-monitor_daq_log "Target device 9a02571e8f05 test hold running"
+monitor_daq_log "Send device result $FAUX_MAC_PREFIX:8f:01 passed"
```

Comment:
similar to other commend, generally like to avoid parameterizing tests unless absolutely necessary. Can all the FOT tests just be run with a fixed alternate prefix, avoiding the need to parameterize here?